packages/tab/src/index.js

```diff
@@ -0,0 +1,3 @@
+import Tabs from './Tabs'

```

Comment:
Missing ";" at the end of the line. This might fail the build while building on DEV.